emerlin2caom2/main_app.py

```diff
@@ -4,46 +4,83 @@
 
 from caom2 import SimpleObservation, ObservationIntentType, Target, Telescope, TypedOrderedDict, Plane, Artifact, \
     ReleaseType, ObservationWriter, ProductType, ChecksumURI, Provenance, Position, Point, Energy, TargetPosition, \
-    Interval, TypedSet, Polarization, shape
+    Interval, TypedSet, Polarization, shape, Proposal, Instrument, DerivedObservation
+from pkg_resources import Environment
 
 import casa_reader as casa
 import measurement_set_metadata as msmd
 import fits_reader as fr
-import settings_file as set
+import settings_file as set_f
+import math
 
 __all__ = [
     'EmerlinMetadata',
+    'polar2cart',
+    'emcp2dict'
 ]
 
+
+def polar2cart(r, theta, phi):
```

Comment:
Due to my lack of pythonic knowledge, is it more standard to place "little" functions at the top, or the bottom, of the python package?  Or does it not matter in python?  I had previously placed them in casa_reader (i.e. freq to wl fcn) as that was where data were coming from so I could just convert them there, but maybe that is not the way to go?

Reply:
On this same idea, the notes say that this version converts wavelength to frequency, but caom accepts wavelength as a standard, and I cannot find any places in this code that do convert wavelength to frequency.  I did find a new addition going the opposite direction, which may be redundant code as there is a mini-function doing this in casa_reader.py.  Do we want to move the casa_reader all to in-line conversions like the new one?  Or consolidate somehow?       ` plane.energy.restwav = 3e8/fits_header_data['central_freq']  # change freq to wav and check against model`

Reply:
Putting it in the casa reader is probably cleaner, I change this too.